target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-c2600.dts

```diff
@@ -27,6 +27,12 @@
 
 	chosen {
 		linux,stdout-path = "serial0:115200n8";
+		leds {
+			boot = &power;
+			failsafe = &ledgnr;
+			running = &power;
+			upgrade= &ledgnr;
+		};
 	};
```

Comment:
Huh? Have you changed your mind about this? We have the same in lantiq since https://github.com/lede-project/source/commit/a3042500ab1114bae29ae55071aa3f26fd19c0a4.